CHANGELOG.md

```diff
@@ -5,8 +5,13 @@ All notable, unreleased changes to this project will be documented in this file.
 # 3.12.0 [Unreleased]
 
 ### Breaking changes
+- `stocks` and `channelListings` inputs for preview `ProductVariantBulkUpdate` mutation has been changed. Both inputs have been extended by:
+    - `create` input - list of items that should be created
+    - `update` input - list of items that should be updated
+    - `remove` input - list of objects ID's that should be removed
```

Comment:
How do I migrate from 3.11 to 3.12 if I was using this API? We should include something like "if your platform relies on this Preview feature, make sure you update your mutations from `foo(bar: baz)` to `foo(xyz: baz)`."